README.md

```diff
@@ -232,7 +232,7 @@ Start the service. Enable will make it startup on reboot.
 systemctl enable -q --now --user autobrr@$USER
 ```
 
-By default it the config is set to listen on only `127.0.0.1`.  It's highly advised to put it behind a reverse-proxy like nginx or traefik etc.
+By default, the configuration is set to listen on `127.0.0.1`. It is highly recommended to use a reverse proxy like [nginx](https://autobrr.com/installation/linux#nginx), [caddy](https://autobrr.com/installation/linux#caddy) or [traefik](https://autobrr.com/installation/docker#traefik).
```

Comment:
Does an end user gain anything by setting up the entirety of nginx, caddy, or traefik for autobrr usage only? Wouldn't it be better to say it's supported, as opposed to implying that not running autobrr in this fashion is a mistake.

Reply:
Made a slight modification.